app/src/main/java/org/mozilla/focus/open/AppAdapter.java

```diff
@@ -13,15 +12,13 @@
 import android.view.View;
 import android.view.ViewGroup;
 
-import org.mozilla.focus.R;
-
 import java.util.ArrayList;
 import java.util.Collections;
 import java.util.Comparator;
 import java.util.List;
 
-public class AppAdapter extends RecyclerView.Adapter<AppViewHolder> {
-    /* package-private */ static class App {
+public class AppAdapter extends RecyclerView.Adapter<RecyclerView.ViewHolder> {
```

Comment:
\[PMD\] INFO: The class 'AppAdapter' has a Cyclomatic Complexity of 3 \(Highest = 5\)\.